onboarding-manager/internal/tinkerbell/bootstrap.go

```diff
@@ -0,0 +1,52 @@
+// SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+package tinkerbell
+
+import (
+	"github.com/open-edge-platform/infra-onboarding/onboarding-manager/internal/env"
+	"github.com/open-edge-platform/infra-onboarding/onboarding-manager/internal/tinkerbell/templates"
+)
+
+func Bootstrap() error {
+	zlog.Info().Msg("Bootstrapping Tinkerbell state")
+
+	if err := clearAllTemplates(); err != nil {
```

Comment:
Even after the OM service restarts, will it clear the templates and create new ones, irrespective of workflow status?

Reply:
See my other comment